libmscore/textlinebase.cpp

```diff
@@ -382,6 +382,8 @@ void TextLineBaseSegment::layout()
 
 void TextLineBaseSegment::spatiumChanged(qreal ov, qreal nv)
       {
+      SpannerSegment::spatiumChanged(ov, nv);
```

Comment:
It is better to call `LineSegment::spatiumChanged()` here (as `LineSegment` is a direct base of `TextLineBaseSegement`) in case `LineSegment` eventually gets its own override implemented.

Reply:
Oh, sorry, my bad.